### ISSUE TITLE
Migrate to memcached@.service

### ISSUE THREAD

**Issue by jernst** [decko-commons/decko-ubos#19]
... once uboslinux/ubos-packages#30 is closed.

**Comment by ethn** [comment]
That's closed.  Do I need to do anything on this?

**Comment by ethn** [comment]
@jernst are there still todos here?

**Comment by jernst** [comment]
Yes, but not critical only "cleanup". Instead of using its own .service file, decko should now use the default memcached@.service file that hadn't existed at the time.
